lib/modules/datasource/nuget/v3.ts

```diff
@@ -131,7 +154,7 @@ export async function getReleases(
       if (releaseTimestamp) {
         release.releaseTimestamp = releaseTimestamp;
       }
-      if (semver.valid(version) && !semver.prerelease(version)) {
+      if (versioning.isValid(version) && versioning.isStable(version)) {
```

Comment:
why not simply check here if version is grater than current latest version? so we don't have to do the expensive sorting here.

Reply:
Only that I was trying to make minimal changes to this function overall. There's a fallback later in the function that assumes the last element in the array is the most recent unstable version, in the case there are no stable versions yet.  Sorting first meant I didn't have to rearrange most of this function to capture both versions in a single linear scan.

You're right though, it would be more efficient. I'll do that instead :)